WordPress/src/main/res/drawable/post_detail_avatar_internal_border.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:bottom="@dimen/avatar_background_padding"
+        android:left="@dimen/avatar_background_padding"
+        android:right="@dimen/avatar_background_padding"
+        android:top="@dimen/avatar_background_padding">
+
+        <shape android:shape="oval">
+            <stroke
+                android:width="0.5dp"
```

Comment:
IIRC using decimals for `dp` either doesn't work (it's rounded to `1dp`) or it's not recommended. In other places where the design specs stated we should use `0.5dp` (such as [here](https://github.com/wordpress-mobile/WordPress-Android/pull/19081#issuecomment-1698351129)) I ended up using `1dp` instead.

Have you heard of this before? Maybe I'm wrong, but wanted to bring it up.

Reply:
AFAIK using decimal values is allowed and goes through the normal calculation/conversion to `px` to be drawn, ensuring that it is at least `1px` (hairline). I cannot find right now the code for that or where I learned that from, but it works, at least in XML layouts (which is the case). 

For Compose (which does not apply here but just adding) I'm not entirely sure if they are doing the same logic, but since they explicitly set `Dp.value` as Float, I believe they also support decimal `dp` values there.

Since we do actually want 0.5dp, so it's smaller than the 1dp border (which is visible in the screenshots for higher DPI devices), and also depending on DPI it might be more than 1px, I think it makes sense to use the decimal value in `dp` instead of using `1dp` or a `px` value.

Wdyt?